src/main/scala/ch/epfl/bluebrain/nexus/kg/async/DistributedCache.scala

```diff
@@ -21,9 +21,10 @@ import monix.eval.Task
 import scala.concurrent.{ExecutionContext, Future}
 
 /**
-  * Project specific operations.
+  * Contract for the distributed cache that keeps in-memory metadata and content
+  * of the resources indexed by the service.
   */
-trait Projects[F[_]] {
+trait DistributedCache[F[_]] {
```

Comment:
It is not a big deal, but I considered this trait as `bundle operations for Projects`, as an analogy to what `Resources` trait is for any generic resource. 

Calling it `DistributedData` is only exposing information about the technique we use to deal with the data, but not about what type of data/operation this trait takes care of.

Reply:
If it contains operations related to projects only we should preserve its name, otherwise if it includes things outside the realm of projects then Cache is probably a better name.